configuration/developing-community-health-applications.md

```diff
@@ -565,10 +566,18 @@ An array of cards to show below the summary on the profile page. Each card has i
 | property | type | description | required |
 |---|---|---|--|
 | `label` | `translation key` | Label on top of card | yes |
-| `fields` | Array of `Fields` | The content of the card | yes |
-| `appliesToType` | Array of contact types | Only calls `appliesIf` if the contact is a match. For example, `['person']`. | no |
-| `appliesIf` | boolean | True if the field should be shown. | no |
-| `modifyContext` | `function(context)` | Used to modify or add values of the `context` | no |
+| `appliesToType` | Array of contact types `string` | Only calls `appliesIf` if the contact's type matches one or more of the array elements. For example, `['person']`. | no |
+| `appliesIf` | `boolean` | True if the field should be shown. | no |
+| `modifyContext` | `function(context)` | Used to modify or add values of the data which is passed as input to forms filled from the contact page | no |
+| `fields` | `Array[]` of [fields](#fields) | The content of the card | yes |
+| `fields[n].appliesIf` | `boolean` or `function(report)` | Same as Fields.appliesIf above | |
```

Comment:
When reading the markdown as a reference it is hard to follow by jumping back and forth with the Fields section. I would duplicate the info, and we can still note that this is the same as Fields.appliesIf above. Listing the required fields here is also useful without having to jump around in the doc.